lib/api/html/accessibility.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * This file contains Beans accessibility features.
+ *
+ * @package Beans\Framework\API\HTML
+ *
+ * @since   1.5.0
+ */
+
+/**
+ * Build the skip links.
+ *
+ * @since 1.5.0
+ *
+ * @return void
+ */
+function beans_build_skip_links() {
+	$skip_links = array();
+	$layout     = beans_get_layout();
+
+	if ( has_nav_menu( 'primary' ) ) {
+		$skip_links['beans-primary-navigation'] = __( 'Skip to the primary navigation.', 'tm-beans' );
+	}
+
+	$skip_links['beans-content'] = __( 'Skip to the content.', 'tm-beans' );
+
+	if ( 'c' !== $layout ) {
+
+		if ( in_array( $layout, array( 'c_sp', 'sp_c', 'c_sp_ss', 'sp_c_ss', 'sp_ss_c' ), true ) && beans_is_active_widget_area( 'sidebar_primary' ) ) {
+			$skip_links['beans-primary-sidebar'] = __( 'Skip to the primary sidebar.', 'tm-beans' );
+		}
+
+		if ( in_array( $layout, array( 'c_ss', 'ss_c', 'c_sp_ss', 'sp_c_ss', 'sp_ss_c' ), true ) && beans_is_active_widget_area( 'sidebar_secondary' ) ) {
```

Comment:
Same as Line 29.  

Here's the new function that goes at the end of the `api/layout/functions.php`:
```
/**
 * Check if the given layout has a secondary sidebar.
 *
 * @since 1.5.0
 *
 * @param string $layout The layout to check.
 *
 * @return bool
 */
function beans_has_secondary_sidebar( $layout ) {

	if ( ! in_array( $layout, array( 'c_ss', 'ss_c', 'c_sp_ss', 'sp_c_ss', 'sp_ss_c' ), true ) ) {
		return false;
	}

	return beans_is_active_widget_area( 'sidebar_secondary' );
}
```

And then here on Line 33, let's use it like this:

```
if ( beans_has_secondary_sidebar( $layout ) ) {
```